### PR TITLE
Upgrade Rubocop

### PR DESCRIPTION
**Pull request by jwswj**
Resolving https://nvd.nist.gov/vuln/detail/CVE-2017-8418

/cc @grosser

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ gem "mime-types", "~> 2.99", :platforms => [:ruby_19, :jruby]
 gem "bump"
 gem "rake"
 gem "rspec"
-gem "rubocop", "0.39.0"
+gem "rubocop", ">= 0.49.0"
```
Comment (grosser) [inline Gemfile]: this should be `~> 0.64.0` so it does not randomly update and breaks

Comment (jwswj) [reply Gemfile]: Updated.

**Comment by grosser** [comment]
this needs travis ... bet it's completely broken :D

**Comment by ggrossman** [comment]
Rebased on master to get Travis working.

**Comment by jwswj** [comment]
Looks good @grosser @ggrossman

**Comment by grosser** [comment]
ok to get back on green, but someone should kill the todos :) <-> make  new jira for that

**Comment by jwswj** [comment]
Yeh @grosser , not actually "good". Looks not so good, but happy to move forward.

**Pull request merged**